centinel/primitives/tcpdump.py

```diff
@@ -10,21 +10,39 @@
 
 
 # local imports
+import centinel
 from centinel import command
 
 
 class Tcpdump():
     """Class to interface between tcpdump and Python"""
 
-    def __init__(self, filename=None):
+    def __init__(self, filename=None, pcap_args=None):
         if filename is None:
             temp_file = tempfile.NamedTemporaryFile(mode='w', delete=False)
             temp_file.close()
             filename = temp_file.name
         self.filename = filename
+        # don't change this to default value because it is a mutable
+        # type and whatever you change it to will become the new
+        # default value until the interpreter is restarted
+        if pcap_args is None:
+            # use the centinel configured tcpdump options if available
+            # (if not specified by the user, this will be -i any, so
+            # the same as below
+            if hasattr(centinel.conf['experiments'], 'tcpdump_params'):
+                pcap_args = centinel.conf['experiments']['tcpdump_params']
```

Comment:
A few lines below, it's an array extension that merges two arrays. We are passing these arguments to subprocess, which can handle either a string of args or an array of args. You can mix and match the two together and I verified that it correctly collected a pcap for the baseline linear test, so I know that it works.